packages/lib/server-only/admin/get-recipients.ts

```diff
@@ -0,0 +1,20 @@
+import { prisma } from '@documenso/prisma';
```

Comment:
I'd also change the filename to `get-recipient-stats` and align the method name to that 🙌